projects/packages/masterbar/src/class-main.php

```diff
@@ -27,21 +27,25 @@ public static function init() {
 			return;
 		}
 
+		$host                    = new Host();
 		$should_use_nav_redesign = function_exists( 'wpcom_is_nav_redesign_enabled' ) && wpcom_is_nav_redesign_enabled();
 
-		if ( ! $should_use_nav_redesign ) {
+		if ( ! $should_use_nav_redesign && ! $host->is_wpcom_simple() ) {
 			new Masterbar();
 		}
 
 		new Admin_Color_Schemes();
 
-		if ( ( new Host() )->is_woa_site() ) {
+		if ( $host->is_wpcom_platform() ) {
 			new Inline_Help();
 			require_once __DIR__ . '/wp-posts-list/bootstrap.php';
-			require_once __DIR__ . '/profile-edit/bootstrap.php';
 			require_once __DIR__ . '/nudges/bootstrap.php';
 		}
```

Comment:
Something I just noted, should we keep the WoA check here? I see Simple sites are already loading these features in their specific mu-plugin:

fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Sznfgreone.cuc%3Se%3Qns1125q9%23272%2Q291-og

They tested well for me though, so it seems that loading twice these feature don't affect the end result, but maybe we should play safe?

Reply:
Oh never mind, I realized we still don't load this package on Simple sites. We're just making the package ready to make the switch from the mu-plugin to the package.